File: src-model/model/celestials/Planet.java
package model.celestials;


import model.celestials.parts.Orbit;
import model.celestials.parts.Sphere;

public class Planet extends CelestialBody {
	
	public Planet(String name, Orbit orbit, Sphere sphere) {
		super(name, orbit, sphere, CelestialBodyType.PLANET);
	}

}

File: src-model/model/celestials/Planet.kt
package model.celestials

import model.celestials.parts.Clouds
import model.celestials.parts.Orbit
import model.celestials.parts.Ring
import model.celestials.parts.Sphere

class Planet(
        name: String,
        orbit: Orbit,
        clouds: Clouds,
        ring: Ring,
        sphere: Sphere
) : CelestialBody(name, orbit, sphere, clouds, ring)
